proto/kava/auction/v1beta1/auction.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+package kava.auction.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/kava-labs/kava/x/auction/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// BaseAuction defines common attributes of all auctions
+message BaseAuction {
+  option (cosmos_proto.implements_interface) = "Auction";
+
+  uint64 id = 1;
```

Comment:
I believe we are renaming names with ids, so to align with other modules
```suggestion
  uint64 id = 1  [(gogoproto.customname) = "ID"];;
```

Reply:
https://github.com/Kava-Labs/kava/pull/1052/commits/4aa7dba5abcd7ad22fc31df01acea558c7af53f6